CVLByExamples/ReadOnlyReentrancy/README.md

```diff
@@ -0,0 +1,15 @@
+This directory contains two versions of a bank contract and a spec that applies the readOnlyReentrancy 
```

Comment:
We need to put a title to reach readme

Reply:
I don't understand which title and where to put it

Reply:
done